site/phpstan-baseline.neon

```diff
@@ -13086,6 +13086,11 @@ parameters:
 			count: 1
 			path: app/views/forum/ForumThreadView.php
 
+		-
```

Comment:
Instead of adding a new phpstan ignore'd error it would be better to just annotate the function signature with types,
```
public function showAlteredDisplayList($threads, $filtering, $thread_id, $categories_ids, bool $is_full_page, $ajax = false)
```

You can add the annotations to other parameters as well, but ideally the new parameter gets a type annotation so we're slowly improving the codebase quality